logbook-json/src/main/java/org/zalando/logbook/json/FastJsonHttpLogFormatter.java

```diff
@@ -11,6 +11,7 @@
 import org.zalando.logbook.HttpRequest;
 import org.zalando.logbook.HttpResponse;
 import org.zalando.logbook.Precorrelation;
+import org.zalando.logbook.internal.JsonMediaType;
```

Comment:
(Same suggestion for other parts; I just looked ahead and found there are several more, so I stopped here 🙂 )

```suggestion
import org.zalando.logbook.json.internal.JsonMediaType;
```